### DISCUSSION TITLE
Exit insert mode takes me to the begining of the line

### DISCUSSION THREAD

**Discussion by rodrii006** [neovim/neovim#29987] [Q&A]
Hi, I would like my Nvim not to take me to the begining of the line when I exit insert mode. I want the cursor to stay at the position where I exit insert mode. 

How can I do it?

Thanks for your help!

**Comment by bew** [comment]
Make sure you can reproduce the issue without configuration, this doesn't happen for me with `nvim --clean`.
If you can't it means it's a problem with your own configuration of nvim and you should try to debug it yourself and maybe ask on stackoverflow/reddit with relevant part of your config.

**Comment by Hamza12700** [comment]
Maybe your keybinds are conflicting with each other and causing this behavior. Take a look at your config.
